strings/c-or-cpp/strtok.cpp

```diff
@@ -0,0 +1,98 @@
+/* -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.
+
+* File Name : strtok.cpp
+
+* Purpose :
+
+* Creation Date : 26-01-2021
+
+* Last Modified : Fri Feb  5 18:41:14 2021
+
+* Created By : Kumar Karan
+
+* Problem Description :
+
+_._._._._._._._._._._._._._._._._._._._._.*/
+
```

Comment:
Please delete theses line, we're git for manage the history of file

Reply:
Ok thanks Ming, done !